QModManager/OptionsManager.cs

```diff
@@ -60,7 +65,242 @@ internal static void Postfix(uGUI_OptionsPanel __instance)
                     AddToggleOption.Invoke(__instance, new object[] { ModsTab, "Enable debug logs", Config.EnableDebugLogs, new UnityAction<bool>(value => Config.EnableDebugLogs = value), null });
                     AddToggleOption.Invoke(__instance, new object[] { ModsTab, "Enable developer mode", Config.EnableDevMode, new UnityAction<bool>(value => Config.EnableDevMode = value), null });
                 }
+                #endregion Mod Config
+
+                #region Mod List
```

Comment:
@desperationfighter Please make it optional.

Reply:
@zorgesho please explain your reasoning on this request.

Reply:
continued on discord